openlibrary/templates/borrow_admin.html

```diff
@@ -212,6 +212,17 @@ <h3>Waiting List</h3>
         <h3>You</h3>
         <ul>
             <li>You are <a href="$ctx.user.key" title="$ctx.user.key">$ctx.user.displayname</a></li>
+            <li>Your IP is <span class="small">$user_ip</span></li>
```

Comment:
This was removed recently, you'll need to rebase to get the latest version of this file.